lib/json2csv.js

```diff
@@ -137,6 +137,14 @@ function checkParams(params) {
 
   //#check include empty rows, defaults to false
   params.includeEmptyRows = params.includeEmptyRows || false;
+
+  //#check unwindPath, defaults to empty array
+  params.unwindPath = params.unwindPath || [];
+
+  // if unwindPath is not in array [{}], then just create 1 item array.
+  if (!Array.isArray(params.unwindPath)) {
```

Comment:
Actually this part of code, make the change compatible with old single string parameter. I have not extensively tested, but it should work fine. So I would say no breaking changes.